plugin/latexlivepreview.vim

```diff
@@ -184,14 +184,23 @@ EEOOFF
                         \ l:tmp_root_dir . '/' . bib_fn)                                " TODO: may fail if same bibfile names in different dirs
         endfor
 
+        if exists('g:livepreview_biber') && g:livepreview_biber > 0
+            let s:bibexec = 'biber *.bcf'
+        else
+            let s:bibexec = 'bibtex *.aux'
+        endif
+
         " Update compile command with bibliography
         let b:livepreview_buf_data['run_cmd'] =
                 \       'env ' .
                 \               'TEXMFOUTPUT=' . l:tmp_root_dir . ' ' .
                 \               'TEXINPUTS=' . l:tmp_root_dir
                 \                            . ':' . b:livepreview_buf_data['root_dir']
                 \                            . ': ' .
-                \       'bibtex ' . l:tmp_root_dir . '/*.aux' .
+                \ ' && ' .
+                \       b:livepreview_buf_data['run_cmd'] .
```

Comment:
The inflight `run_cmd` is already being called once previous to this point in code, and it is called again a few lines below, do we need it here too ?

https://github.com/xuhdev/vim-latex-live-preview/blob/855c30915547c7e9c19b7e6efb528730c071d2e8/plugin/latexlivepreview.vim#L171